# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/glibc/glibc23/gcc-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./sysdeps/i386/i686/memcmp.S.orig	2005-02-10 15:32:56.000000000 +0200
+++ ./sysdeps/i386/i686/memcmp.S	2005-02-10 15:47:35.000000000 +0200
@@ -351,7 +351,6 @@
 	popl	%esi
 	RETURN
 
-	.section	.rodata
 	ALIGN (2)
 L(table_32bytes) :
 	.long	L(0bytes) - . + 0x0
